test/utils/common.go

```diff
@@ -4,33 +4,28 @@ import (
 	"testing"
 )
 
-func TestCommon(t *testing.T, params *TestParams) {
-	if len(params.DefaultServicePort) > 0 {
-		for _, port := range params.DefaultServicePort {
-			WaitServiceOnline(t, 60, port)
-		}
-	}
-
-	if params.TestEnvConfig == true {
-		SetEnvConfig(t, params.Snap, params.App, params.DefaultServicePort[0])
+func TestCommon(t *testing.T, params Params) {
+	// config tests
+	if params.TestEnvConfig {
+		SetEnvConfig(t, params.Snap, params.App, params.DefaultServicePort)
 	}
-	if params.TestAppConfig == true {
-		SetAppConfig(t, params.Snap, params.App, params.DefaultServicePort[0])
+	if params.TestAppConfig {
+		SetAppConfig(t, params.Snap, params.App, params.DefaultServicePort)
 	}
-	if params.TestGlobalConfig == true {
-		SetGlobalConfig(t, params.Snap, params.App, params.DefaultServicePort[0])
+	if params.TestGlobalConfig {
+		SetGlobalConfig(t, params.Snap, params.App, params.DefaultServicePort)
 	}
-	if params.TestMixedConfig == true {
-		SetMixedConfig(t, params.Snap, params.App, params.DefaultServicePort[0])
+	if params.TestMixedConfig {
+		SetMixedConfig(t, params.Snap, params.App, params.DefaultServicePort)
 	}
 
+	// networking tests
 	if len(params.TestOpenPorts) > 0 {
 		for _, port := range params.TestOpenPorts {
 			WaitServiceOnline(t, 60, port)
 		}
 	}
```

Comment:
Is this subtest redundant with [utils.WaitServiceOnline](https://github.com/canonical/edgex-snap-testing/blob/7e2db2ca2ca71bfe66e66f2a304e0f6d6b67f0bb/test/suites/edgexfoundry/main_test.go#L55)?